GitCommands/CommitDataManager.cs

```diff
@@ -66,18 +66,10 @@ public void UpdateBody(CommitData commitData, bool appendNotesOnly, out string?
             }
             else
             {
-                // $ git log --pretty="format:%B%nNotes:%n%-N" -1
-                // Remove redundant parameter
-                //
-                // The sha1 parameter must match CommitData.Guid.
-                // There's no point passing it. It only creates opportunity for bugs.
-                //
-                // Notes:
-
-                var lines = data.Split(Delimiters.LineFeed);
+                string[] lines = data.Split(Delimiters.LineFeed);
 
                 // Commit message is not re-encoded by Git when format is given
-                commitData.Body = GetModule().ReEncodeCommitMessage(ProcessDiffNotes(startIndex: 0, lines));
+                commitData.Body = GetModule().ReEncodeCommitMessage(ProcessDiffNotes(startIndex: 0, lines)).Replace('\v', '\n');
```

Comment:
Is there a reason to not execute `.Replace('\v', '\n')` inside `ReEncodeCommitMessage`?

Reply:
I doubt the \v is encoded, but it is no question if done outside.